vembrane/ann_types.py

```diff
@@ -49,6 +49,13 @@ def __repr__(self):
     def __hash__(self) -> int:
         return super().__hash__()
 
+    def __getattr__(self, item):
+        print(
+            f"Warning: attribute '{item}' does not exist, returning NA instead.",
```

Comment:
Maybe add more context here, using `self`? And maybe add suggestions, what might be wrong. For example:

* "Annotation '{self}' does not exist in this particular record?"
* "Maybe you mistyped the intended attribute, did you really mean '{item}'?"

Reply:
`NoValue` is a general value-is-missing indicator, so this is not specific to ANN, and `self` is always `NoValue`.
To fix this only for specific ANN types, we'd have to choose a different route, not quite sure what a good solution would be.